.yarnrc

```diff
@@ -2,8 +2,8 @@
 # yarn lockfile v1
 
 
-version v1.19.0
-lastUpdateCheck 1597082923381
+version "1.22.10"
```

Comment:
is this supposed to match up with the release version in `yarn-path`?

Reply:
hmm yeah, this was auto generated and I set the policy to use 1.22.17. Not sure why it says 1.22.10...

Reply:
gotcha, I guess it should be fine since it was autogenerated and still the same minor version